web/Controllers/LandlordController.cs

```diff
@@ -319,16 +324,16 @@ public async Task<IActionResult> TieUserWithLandlord(string inviteLink)
         }
     }
 
-    [HttpGet("{id:int}/dashboard")]
-    public async Task<ActionResult> Dashboard([FromRoute] int id)
+    [HttpGet("dashboard")]
+    public async Task<ActionResult> Dashboard()
     {
         var user = CurrentUser;
-        if (user.LandlordId != id && !user.IsAdmin)
+        if (!user.LandlordId.HasValue)

```

Comment:
what about for admins?

Reply:
This is for the landlord dashboard which I figured admins shouldn't ever need to access (because they have their own dashboard and other pages for dealing with landlord profiles). For some reason the existing code allowed admins to see the dashboard of any landlord, but I'm not sure that makes sense to do, and we never made use of it anyway.

Reply:
Sounds reasoning :)